src/webgpu/web_platform/reftests/README.txt

```diff
@@ -8,7 +8,6 @@ This tests things like:
 - The canvas renders with the correct transfer function.
 - The canvas blends and interpolates in the correct color encoding.
 
-TODO(#915): Test all possible output texture formats (currently only needs rgba16float).
```

Comment:
we are going to want to expand the canvas_complex tests as well (actually they need rgba8unorm tests too - I forgot before). Much less urgent, of course, but we should keep tracking a TODO here.
`TODO(#915): canvas_complex: test rgba8unorm and rgba16float`

Reply:
Sounds fair.